### ISSUE TITLE
#import "RNLine-Swift.h" is Not Found.

### ISSUE THREAD

**Issue by leegun** [xmartlabs/react-native-line#124]
I'm implementing it based on [using-objective-c](https://github.com/xmartlabs/react-native-line?tab=readme-ov-file#using-objective-c), but

#import "RNLine-Swift.h"

is Not Found.

```
yarn add @xmartlabs/react-native-line
cd ios/
pod install
```

has already been done.

**Comment by ardasatata** [comment]
have you checked your pods?

even better if you provide a repo @leegun

**Comment by ardasatata** [comment]
@aravi365 did you use expo by any chance?

**Comment by aravi365** [comment]
> [@aravi365](https://github.com/aravi365) did you use expo by any chance?

No. Using plain react native only as this is an old project but upgraded to latest RN. Looks like this module is not usable btw

**Comment by Joaguirrem** [comment]
@leegun we recently published new versions where we updated the code, fixed bugs, and added new features. Please try using the latest version and let us know if you still encounter any issues!

**Comment by ThangNV1205** [comment]
@leegun Does your project working with version 4.1.0?
